.github/workflows/cpp.yml

```diff
@@ -296,7 +296,7 @@ jobs:
       ARROW_SIMD_LEVEL: ${{ matrix.simd-level }}
       ARROW_SUBSTRAIT: ON
       ARROW_USE_GLOG: OFF
-      ARROW_VERBOSE_THIRDPARTY_BUILD: OFF
+      ARROW_VERBOSE_THIRDPARTY_BUILD: ON
```

Comment:
I assume you'll revert these before merge?

Reply:
Yes, this was to debug the build/link errors on Windows which are now fixed.